ui/src/dashboards/actions/index.js

```diff
@@ -111,65 +113,92 @@ export const editCellQueryStatus = (queryID, status) => ({
   },
 })
 
+export const templateVariableSelected = (dashboardID, templateID, values) => ({
+  type: TEMPLATE_VARIABLE_SELECTED,
+  payload: {
+    dashboardID,
+    templateID,
+    values,
+  },
+})
+
+export const editTemplateVariableSuccess = (dashboardID, data) => ({
```

Comment:
Obsolete code, actually. We're now batch-updating templates via `putDashboard`. Will remove.